src/ucp/api/ucp.h

```diff
@@ -538,6 +538,22 @@ void ucp_worker_destroy(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_CONTEXT
+ * @bried Print information about protocols that would be used by a worker.
+ *
+ * This routine prints information about the protocols being used, thresholds,
+ * UCT methods, and so on, used by the current context.
```

Comment:
"UCT methods" -> transport methods
" and so on, used by the current context" -> " and other useful information associated with the worker"